### PR TITLE
Add report template for integr8ly/integreatly-operator

### PR DESCRIPTION
**Pull request by matskiv**
Hi, I am from RHMI / Integreatly team.
We would like to customize test report comment for our integr8ly/integreatly-operator repo.
We are trying to cut down on flakes in our tests + track PROW issues that we encounter, and this message on test failures would help with that.

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
@matskiv you can fix the failing test by running `make prow-config` and committing the generated changes.

**Comment by matskiv** [comment]
Thanks @laurafitzgerald 
Hopefully it'll pass now 🤞  :)

**Comment by matskiv** [comment]
ping @stevekuznetsov @petr-muller @droslean do you have a minute to review this?

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -496,6 +496,9 @@ plank:
       by [linking to](https://github.com/kubernetes/community/blob/master/contributors/devel/flaky-tests.md#filing-issues-for-flaky-tests)
       an [open issue](https://github.com/{{.Spec.Refs.Org}}/{{.Spec.Refs.Repo}}/issues?q=is:issue+is:open)
       when you hit one in your PR.'
+    integr8ly/integreatly-operator: |-
+      ### Report flaky E2E tests and PROW issues in [INTLY-7197](https://issues.redhat.com/browse/INTLY-7197) epic. Detailed instructions can be found in the description of said epic.
```
Comment (droslean) [inline core-services/prow/02_config/_config.yaml:500]: I don't think that those `###` are necessary at the beginning of the comment. But this is intentional, I fine with it.

Comment (matskiv) [reply core-services/prow/02_config/_config.yaml:500]: That is intentional to make the text bigger. :)

**Pull request merged**